.github/workflows/test.yml

```diff
@@ -6,10 +6,36 @@ on:
     types: [opened, synchronize]
     branches-ignore: [staging, production]
 
+env:
+  # Number of parallel jobs for jest tests
+  CHUNKS: 3
 jobs:
+  config:
+    runs-on: ubuntu-latest
+    name: Define matrix parameters
+    outputs:
+      MATRIX: ${{ steps.set-matrix.outputs.MATRIX }}
+    steps:
+      - name: Set Matrix
+        id: set-matrix
+        uses: actions/github-script@v6
+        with:
+          # Generate matrix array i.e. [0, 1, 2, ...., CHUNKS] for test job
+          script: |
+            core.setOutput('MATRIX', Array.from({ length: process.env.CHUNKS + 1 }, (v, i) => i));
```

Comment:
`process.env.CHUNKS` is the string to due to concatenation it will give 31 chunks answer to my previous question.

Reply:
I will fix this

Reply:
@parasharrajat Fixed and pushed code. Kindly have a look

Reply:
Thanks. let's discuss the other thread.